src/store/PapersContext.tsx

```diff
@@ -115,28 +147,28 @@ export class PapersContextProvider extends Component {
   }
 
   async componentDidMount() {
-    const { avatar, ...profile } = this.state.profile
+    const profile = this.state.profile
     console.log('Papers mounted:', profile)
     await this.tryToReconnect()
 
-    await PapersSound.init(profile.settings.sound)
+    await PapersSound.init(profile?.settings.sound)
   }
 
   componentWillUnmount() {
-    this.state.socket && this.state.socket.offAll()
+    this.state.socket && this.state.socket?.offAll()
 
     if (__DEV__)
       console.log(`
-::::::::::::::::::::::::::::::
-  ... Refreshing app ...
-:::::::::::::::::::::::::::::`)
+        ::::::::::::::::::::::::::::::
```

Comment:
nit: This is different from before. Now the string as literally these white spaces in the console log. I remember that it was ugly in the devs tools, that's why I had it was before. The alternative to keep the indentation is to contact multiple normal strings with `+ \n` at the end of each line.

Reply:
I don't have a big preference but tbh the white space there kinda helps identifying this "special" log IMO, wdyt?

<img width="397" alt="Screenshot 2023-06-17 at 17 01 33" src="https://github.com/cratera/papers-game/assets/63152281/7eee4d8a-6c1c-40a9-abf3-bdfa21102640">

Reply:
And you are debugging a wide dev tools. Back then I had 4 browsers opened simultaneously side by side, each one was only ~250px wide, so every white space counted ehehe

Reply:
Ah I see, then I'll revert it because at some point I'll have to do the same x)

Reply:
Fixed in [b7715be](https://github.com/cratera/papers-game/pull/33/commits/b7715be63c7e79744aa4edb3796598866fc35c04)